PowerEditor/src/Notepad_plus.cpp

```diff
@@ -2567,8 +2567,11 @@ void Notepad_plus::checkClipboard()
 {
 	bool hasSelection = _pEditView->hasSelection();
 	bool canPaste = (_pEditView->execute(SCI_CANPASTE) != 0);
+	//enableCommand(IDM_EDIT_CUT, hasSelection, MENU | TOOLBAR);
+	//enableCommand(IDM_EDIT_COPY, hasSelection, MENU | TOOLBAR);

```

Comment:
DO NOT disable Copy & Cut commands while no selection for not only Copy & Cut entire line, but also letting the shortcuts can trigger  IDM_EDIT_CUT & IDM_EDIT_CUT for non main edit zones (text field of some plugins).